backup: describe how to restore (#31715)
home-assistant/home-assistant.github.io: source/_integrations/backup.markdown

      alias: "Create backup now"
      service: backup.create
```

## Restoring a backup

<div class="note">

> If you use Home Assistant Operating System or Home Assistant Supervised, [restore functionality is already built-in](/common-tasks/os/#restoring-a-backup).

Reviewer: ```suggestion
If you use Home Assistant Operating System or Home Assistant Supervised, [the restore functionality is already built-in](/common-tasks/os/#restoring-a-backup).
```

</div>

> Backups created via the integrations are located in you `/config/backups` directory. Home Assistant Container installation will typically mount this directory via `docker-compose.yml` or `docker run` to a directory of they choice.  

Reviewer: ```suggestion
Backups created via the **Backup** integration are located in your `/config/backups` directory. The Home Assistant Container installation will typically mount this directory via `docker-compose.yml` or `docker run` to a directory of your choice.  
```

> There is currently no builtin way to restore a backup. However, a Home Assistant backup is just a tar file of the `/config` directory, plus some metadata. To manually restore a backup, one can do:

Reviewer: ```suggestion
For Container and Core installations, there is currently no built-in way to restore a backup. However, a Home Assistant backup is just a tar file of the `/config` directory, plus some metadata. To manually restore a backup, you can use the following:
```

```shell

> $ tar -xOf <backup_tar_file> "./homeassistant.tar.gz" | tar --strip-components=1 -zxf - -C <restore_directory>

Reviewer 2: Not sure why our CI passes here, but the leading `$` should not have been added and must be removed.

Can you open up a PR to correct this?

../Frenck

Reviewer: sure, will do! Thank you for the catch!
#31748
